crates/nu-cli/src/commands/autoview/command.rs

```diff
@@ -132,6 +132,12 @@ pub async fn autoview(context: RunnableContext) -> Result<OutputStream, ShellErr
                             out!("{}", s);
                         }
                     }
+                    Value {
+                        value: UntaggedValue::Primitive(Primitive::Nothing),
+                        ..
+                    } => {
+                        // ignore Nothing
+                    }
```

Comment:
@thegedge This is IMO a better solution for the empty InputStream problem.

If you think I should still put the explicit check for `InputStream::is_empty()` in `From<InputStream> for OutputStream`, please let me know and I'll add that as well.

Reply:
tl;dr I think this is something we're going to want to think about more deeply, as I worry there's a lot of nuance. For now, let's revert to what you had before and come back to figure out how to properly describe an empty stream.

---

I preferred the previous code. Before your changes, we would see this:

```
> echo $(ls --long | get target | first)
───┬──
 0 │
───┴──
```

With your changes, we would see nothing at all displayed.

The benefit of what we have right now is that I at least know there's something flowing through the stream. Mind you, it's not great, but it becomes a little harder to debug issues when nothing at all is displayed. Also, we may want to eventually be able to distinguish between "a stream with a single nothing value" and "a stream with no values".

Reply:
I will revert the changes.

I agree this needs addressing in a more thorough way, for example `config clear` now shows the same empty table as output (and it's not a stream with `Primitive::Nothing` in it), which is not nice either and just this doesn't fix it (it can be fixed in config::clear though).

Reply:
Yeah, there's likely a bigger underlying issue here that we should investigate 👍